### PR TITLE
Update submodules

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: 1-trackers/no-js/README.md [hunk 1]
```diff
@@ -12,16 +12,16 @@ The wizard source code can be found [here][wizard]. The logic for generating the
...
+| ![i1][techdocs-image]      | ![i2][setup-image]   |
+| [Technical Docs][techdocs] | [Setup Guide][setup] |
 
-Also see [this introductory blog post][intro-blog-post]. A hosted version of the wizard can be found [on the SnowPlow Analytics website][hosted-wizard].
+Also see [this introductory blog post][intro-blog-post].
 
 ## Copyright and license
 
-The No-JavaScript Tracker is copyright 2012-2013 SnowPlow Analytics Ltd.
+The No-JavaScript Tracker is copyright 2012-2021 SnowPlow Analytics Ltd.
```
Comment (paulboocock) [inline 1-trackers/no-js/README.md:24]: I do like finding a `SnowPlow` reference. It should be `Snowplow` now :-)

File: 5-data-modeling/event-manifest-populator/README.md [hunk 2]
```diff
@@ -39,6 +39,19 @@ Optionally, you can also pass following options:
 * `--jar` to specify S3 path to custom JAR
 
 
+## Duplicate storage configuration JSON
```
[the remark this answers is not in the document]
Comment (adatzer) [reply 5-data-modeling/event-manifest-populator/README.md:42]: Before it was linking to this paragraph of the wiki:
`[dynamodb-config]`: https://github.com/snowplow/snowplow/wiki/Configuring-storage-targets#dynamodb
(also linked in [docs](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/loaders-storage-targets/events-manifest-populator/)).
This block of text is just this paragraph copied here to replace that wiki link.

File: 1-trackers/no-js/README.md (hunk 1 above)
Comment (adatzer) [reply 1-trackers/no-js/README.md:24]: Thank you! I didn't notice! i'll add a commit now to replace them all.

File: 1-trackers/no-js/README.md [hunk 3]
```diff
@@ -35,10 +35,8 @@ limitations under the License.
 [wizard]: https://github.com/snowplow/snowplow/blob/master/1-trackers/no-js-tracker/html/no-js-embed-code-generator.html
 [js-wizard]: https://github.com/snowplow/snowplow/blob/master/1-trackers/no-js-tracker/js/no-js-tracker.js
 [intro-blog-post]: http://snowplowanalytics.com/blog/2013/01/29/introducing-the-no-js-tracker/
```
Comment (benjben) [inline 1-trackers/no-js/README.md:37]: Dead link

File: 1-trackers/no-js/README.md [hunk 4]
```diff
@@ -35,10 +35,8 @@ limitations under the License.
 [wizard]: https://github.com/snowplow/snowplow/blob/master/1-trackers/no-js-tracker/html/no-js-embed-code-generator.html
```
Comment (benjben) [inline 1-trackers/no-js/README.md:35]: Dead link

File: 1-trackers/no-js/README.md [hunk 5]
```diff
@@ -35,10 +35,8 @@ limitations under the License.
...
 [js-wizard]: https://github.com/snowplow/snowplow/blob/master/1-trackers/no-js-tracker/js/no-js-tracker.js
```
Comment (benjben) [inline 1-trackers/no-js/README.md:36]: Dead link

File: 5-data-modeling/event-manifest-populator/README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
```
Comment (benjben) [inline 5-data-modeling/event-manifest-populator/README.md:2]: We should remove what is on this page and link to [our docs website](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/loaders-storage-targets/events-manifest-populator/) instead (and update it if there is information that is on this page but not on docs).

**Review by benjben — APPROVED** [review]
That looks awesome @adatzer !

Out of curiosity, how did you update all the submodules, did you do it manually or have you created an automated way to do so ?

File: 5-data-modeling/event-manifest-populator/README.md (hunk 6 above)
Comment (adatzer) [reply 5-data-modeling/event-manifest-populator/README.md:2]: Thank you very much @benjben ! I will fix these in the "Update READMEs" commit, force push and check the docs for missing info there.
Concerning the submodule updates, i just did it manually.

**Pull request merged**